chapters/data-management-plan.qmd

```diff
@@ -0,0 +1,119 @@
+# Data Management Plan
+
+## What is a DMP
+
+Data Management Plan (DMP) is a document outlining how research data will be handled throughout research lifecycle. A DMP is a structured way to address data collection, organization, storage, sharing, and preservation. It outlines the measures taken to ensure data security. DMP also address how data will be preserved and made available for future use.
```

Comment:
I'm missing some articles here. Could we add:

Line 5: 
- **A** Data Management Plan
- **the** research lifecycle
- DMP > DMP**s** (plural)